.oelint-ruleset.json

```diff
@@ -0,0 +1,250 @@
+{
+  "oelint.append.protvars": "error",
```

Comment:
@m-iwanicki Please sync config from: https://github.com/3mdeb/hooks/pull/4/files and apply necessary changes

Reply:
@macpijan Updated in [e1af86a](https://github.com/3mdeb/meta-trenchboot/pull/28/commits/e1af86a3e06c19ff601d84f4ec7e4c4bb33cba7a)